Documentation/Translating: note new GitHub organisation

m-aciek opened the pull request:
* Adds a mention about the GiHub organisation (added after a discussion on docs community monthly meeting; not sure if that FAQ section is the perfect place for that, but I don't have a better idea).
* Updates link to the translations dashboard (the dashboard has been moved to the organisation).
* Mentions bootstraper project (sibling to the cookiecutter).
* Improves the list of resources needed for language switcher inclusion.

<!--
Thanks for your contribution!
Please read this comment in its entirety. It's quite important.

# Pull Request title

It should describe the change to be made.

Most PRs will require an issue number. Trivial changes, like fixing a typo,
do not need an issue.
-->


<!-- readthedocs-preview cpython-devguide start -->
----
📚 Documentation preview 📚: https://cpython-devguide--1507.org.readthedocs.build/

<!-- readthedocs-preview cpython-devguide end -->

AA-Turner commented:
Could we use this PR to note that the organisation exists in an appropriate place?

StanFromIreland commented on documentation/translating.rst:
```diff
@@ -112,20 +112,21 @@ First subscribe to the `translation mailing list <translation_ml_>`_,
-- Create the GitHub repo (anywhere) with the right hierarchy (using the
-  cookiecutter).
+- Create the GitHub repo (anywhere) with the right hierarchy (using one
+  of bootstrappers).
```
```suggestion
  of the bootstrappers).
```

StanFromIreland commented on documentation/translating.rst:
```diff
@@ -112,20 +112,21 @@ First subscribe to the `translation mailing list <translation_ml_>`_,
 - Ensure we update this page to reflect your work and progress, either via a
   PR or by asking on the `translation mailing list <translation_ml_>`_.
-- When ``bugs.html``, ``tutorial``, and ``library/functions`` are 100%
+- When ``bugs``, ``tutorial``, and ``library/functions`` are 100%
```
```suggestion
- When ``bugs.html``, ``tutorial``, and ``library/functions`` are 100%
```

Bugs is a single file whereas the others are folders - I think there is no reason to change

m-aciek commented:
> Could we use this PR to note that the organisation exists in an appropriate place?

I've added a link in the FAQ tools section.

Also I've added a mention about the second translation repo bootstrapper and added a minor fix in the PEP 545 resources list.
